diff --git a/index.d.ts b/index.d.ts
index 4f975490cbddac887873fb9988de9b7b26567286..4ed7c9c0e42a7c7b59342236d7d4296ba20ef250 100644
--- a/index.d.ts
+++ b/index.d.ts
@@ -1176,7 +1176,7 @@ export class Command<Args extends any[] = [], Opts extends OptionValues = {}> {
   /**
    * Return an object containing merged local and global option values as key-value pairs.
    */
-  optsWithGlobals<T extends OptionValues>(): T;
+  optsWithGlobals<T extends OptionValues>(): T & Opts;
 
   /**
    * Set the description.
