diff --git a/node_modules/@apollo/federation/dist/composition/rules.js b/node_modules/@apollo/federation/dist/composition/rules.js
index 02ee0f8..3a7e98c 100644
--- a/node_modules/@apollo/federation/dist/composition/rules.js
+++ b/node_modules/@apollo/federation/dist/composition/rules.js
@@ -3,17 +3,14 @@ Object.defineProperty(exports, "__esModule", { value: true });
 exports.compositionRules = void 0;
 const specifiedRules_1 = require("graphql/validation/specifiedRules");
 const graphql_1 = require("graphql");
-const UniqueTypeNames_1 = require("graphql/validation/rules/UniqueTypeNames");
-const UniqueEnumValueNames_1 = require("graphql/validation/rules/UniqueEnumValueNames");
-const PossibleTypeExtensions_1 = require("graphql/validation/rules/PossibleTypeExtensions");
-const UniqueFieldDefinitionNames_1 = require("graphql/validation/rules/UniqueFieldDefinitionNames");
+const validation_1 = require("graphql/validation");
 const sdl_1 = require("./validate/sdl");
 const omit = [
-    graphql_1.UniqueDirectivesPerLocationRule,
-    UniqueTypeNames_1.UniqueTypeNames,
-    UniqueEnumValueNames_1.UniqueEnumValueNames,
-    PossibleTypeExtensions_1.PossibleTypeExtensions,
-    UniqueFieldDefinitionNames_1.UniqueFieldDefinitionNames,
+    validation_1.UniqueDirectivesPerLocationRule,
+    validation_1.UniqueTypeNamesRule,
+    validation_1.UniqueEnumValueNamesRule,
+    validation_1.PossibleTypeExtensionsRule,
+    validation_1.UniqueFieldDefinitionNamesRule,
 ];
 exports.compositionRules = specifiedRules_1.specifiedSDLRules
     .filter(rule => !omit.includes(rule))
diff --git a/node_modules/@apollo/federation/dist/csdlDirectives.js b/node_modules/@apollo/federation/dist/csdlDirectives.js
index e086363..8ca3052 100644
--- a/node_modules/@apollo/federation/dist/csdlDirectives.js
+++ b/node_modules/@apollo/federation/dist/csdlDirectives.js
@@ -7,7 +7,7 @@ exports.ComposedGraphDirective = new graphql_1.GraphQLDirective({
     locations: [graphql_1.DirectiveLocation.SCHEMA],
     args: {
         version: {
-            type: graphql_1.GraphQLNonNull(graphql_1.GraphQLInt),
+            type: new graphql_1.GraphQLNonNull(graphql_1.GraphQLInt),
         },
     },
 });
@@ -16,10 +16,10 @@ exports.GraphDirective = new graphql_1.GraphQLDirective({
     locations: [graphql_1.DirectiveLocation.SCHEMA],
     args: {
         name: {
-            type: graphql_1.GraphQLNonNull(graphql_1.GraphQLString),
+            type: new graphql_1.GraphQLNonNull(graphql_1.GraphQLString),
         },
         url: {
-            type: graphql_1.GraphQLNonNull(graphql_1.GraphQLString),
+            type: new graphql_1.GraphQLNonNull(graphql_1.GraphQLString),
         },
     },
     isRepeatable: true,
@@ -29,7 +29,7 @@ exports.OwnerDirective = new graphql_1.GraphQLDirective({
     locations: [graphql_1.DirectiveLocation.OBJECT],
     args: {
         graph: {
-            type: graphql_1.GraphQLNonNull(graphql_1.GraphQLString),
+            type: new graphql_1.GraphQLNonNull(graphql_1.GraphQLString),
         },
     },
 });
@@ -38,10 +38,10 @@ exports.KeyDirective = new graphql_1.GraphQLDirective({
     locations: [graphql_1.DirectiveLocation.OBJECT],
     args: {
         fields: {
-            type: graphql_1.GraphQLNonNull(graphql_1.GraphQLString),
+            type: new graphql_1.GraphQLNonNull(graphql_1.GraphQLString),
         },
         graph: {
-            type: graphql_1.GraphQLNonNull(graphql_1.GraphQLString),
+            type: new graphql_1.GraphQLNonNull(graphql_1.GraphQLString),
         },
     },
     isRepeatable: true,
@@ -51,7 +51,7 @@ exports.ResolveDirective = new graphql_1.GraphQLDirective({
     locations: [graphql_1.DirectiveLocation.FIELD_DEFINITION],
     args: {
         graph: {
-            type: graphql_1.GraphQLNonNull(graphql_1.GraphQLString),
+            type: new graphql_1.GraphQLNonNull(graphql_1.GraphQLString),
         },
     },
 });
@@ -60,7 +60,7 @@ exports.ProvidesDirective = new graphql_1.GraphQLDirective({
     locations: [graphql_1.DirectiveLocation.FIELD_DEFINITION],
     args: {
         fields: {
-            type: graphql_1.GraphQLNonNull(graphql_1.GraphQLString),
+            type: new graphql_1.GraphQLNonNull(graphql_1.GraphQLString),
         },
     },
 });
@@ -69,7 +69,7 @@ exports.RequiresDirective = new graphql_1.GraphQLDirective({
     locations: [graphql_1.DirectiveLocation.FIELD_DEFINITION],
     args: {
         fields: {
-            type: graphql_1.GraphQLNonNull(graphql_1.GraphQLString),
+            type: new graphql_1.GraphQLNonNull(graphql_1.GraphQLString),
         },
     },
 });
diff --git a/node_modules/@apollo/federation/dist/directives.js b/node_modules/@apollo/federation/dist/directives.js
index 90a963a..f5bd3c4 100644
--- a/node_modules/@apollo/federation/dist/directives.js
+++ b/node_modules/@apollo/federation/dist/directives.js
@@ -7,7 +7,7 @@ exports.KeyDirective = new graphql_1.GraphQLDirective({
     locations: [graphql_1.DirectiveLocation.OBJECT, graphql_1.DirectiveLocation.INTERFACE],
     args: {
         fields: {
-            type: graphql_1.GraphQLNonNull(graphql_1.GraphQLString),
+            type: new graphql_1.GraphQLNonNull(graphql_1.GraphQLString),
         },
     },
 });
@@ -24,7 +24,7 @@ exports.RequiresDirective = new graphql_1.GraphQLDirective({
     locations: [graphql_1.DirectiveLocation.FIELD_DEFINITION],
     args: {
         fields: {
-            type: graphql_1.GraphQLNonNull(graphql_1.GraphQLString),
+            type: new graphql_1.GraphQLNonNull(graphql_1.GraphQLString),
         },
     },
 });
@@ -33,7 +33,7 @@ exports.ProvidesDirective = new graphql_1.GraphQLDirective({
     locations: [graphql_1.DirectiveLocation.FIELD_DEFINITION],
     args: {
         fields: {
-            type: graphql_1.GraphQLNonNull(graphql_1.GraphQLString),
+            type: new graphql_1.GraphQLNonNull(graphql_1.GraphQLString),
         },
     },
 });
diff --git a/node_modules/@apollo/federation/src/csdlDirectives.ts b/node_modules/@apollo/federation/src/csdlDirectives.ts
index f8cee39..f48e91d 100644
--- a/node_modules/@apollo/federation/src/csdlDirectives.ts
+++ b/node_modules/@apollo/federation/src/csdlDirectives.ts
@@ -11,7 +11,7 @@ export const ComposedGraphDirective = new GraphQLDirective({
   locations: [DirectiveLocation.SCHEMA],
   args: {
     version: {
-      type: GraphQLNonNull(GraphQLInt),
+      type: new GraphQLNonNull(GraphQLInt),
     },
   },
 });
@@ -21,10 +21,10 @@ export const GraphDirective = new GraphQLDirective({
   locations: [DirectiveLocation.SCHEMA],
   args: {
     name: {
-      type: GraphQLNonNull(GraphQLString),
+      type: new GraphQLNonNull(GraphQLString),
     },
     url: {
-      type: GraphQLNonNull(GraphQLString),
+      type: new GraphQLNonNull(GraphQLString),
     },
   },
   isRepeatable: true,
@@ -35,7 +35,7 @@ export const OwnerDirective = new GraphQLDirective({
   locations: [DirectiveLocation.OBJECT],
   args: {
     graph: {
-      type: GraphQLNonNull(GraphQLString),
+      type: new GraphQLNonNull(GraphQLString),
     },
   },
 });
@@ -45,10 +45,10 @@ export const KeyDirective = new GraphQLDirective({
   locations: [DirectiveLocation.OBJECT],
   args: {
     fields: {
-      type: GraphQLNonNull(GraphQLString),
+      type: new GraphQLNonNull(GraphQLString),
     },
     graph: {
-      type: GraphQLNonNull(GraphQLString),
+      type: new GraphQLNonNull(GraphQLString),
     },
   },
   isRepeatable: true,
@@ -59,7 +59,7 @@ export const ResolveDirective = new GraphQLDirective({
   locations: [DirectiveLocation.FIELD_DEFINITION],
   args: {
     graph: {
-      type: GraphQLNonNull(GraphQLString),
+      type: new GraphQLNonNull(GraphQLString),
     },
   },
 });
@@ -69,7 +69,7 @@ export const ProvidesDirective = new GraphQLDirective({
   locations: [DirectiveLocation.FIELD_DEFINITION],
   args: {
     fields: {
-      type: GraphQLNonNull(GraphQLString),
+      type: new GraphQLNonNull(GraphQLString),
     },
   },
 });
@@ -79,7 +79,7 @@ export const RequiresDirective = new GraphQLDirective({
   locations: [DirectiveLocation.FIELD_DEFINITION],
   args: {
     fields: {
-      type: GraphQLNonNull(GraphQLString),
+      type: new GraphQLNonNull(GraphQLString),
     },
   },
 });
diff --git a/node_modules/@apollo/federation/src/directives.ts b/node_modules/@apollo/federation/src/directives.ts
index dfd723c..60e2ed3 100644
--- a/node_modules/@apollo/federation/src/directives.ts
+++ b/node_modules/@apollo/federation/src/directives.ts
@@ -26,7 +26,7 @@ export const KeyDirective = new GraphQLDirective({
   locations: [DirectiveLocation.OBJECT, DirectiveLocation.INTERFACE],
   args: {
     fields: {
-      type: GraphQLNonNull(GraphQLString),
+      type: new GraphQLNonNull(GraphQLString),
     },
   },
 });
@@ -46,7 +46,7 @@ export const RequiresDirective = new GraphQLDirective({
   locations: [DirectiveLocation.FIELD_DEFINITION],
   args: {
     fields: {
-      type: GraphQLNonNull(GraphQLString),
+      type: new GraphQLNonNull(GraphQLString),
     },
   },
 });
@@ -56,7 +56,7 @@ export const ProvidesDirective = new GraphQLDirective({
   locations: [DirectiveLocation.FIELD_DEFINITION],
   args: {
     fields: {
-      type: GraphQLNonNull(GraphQLString),
+      type: new GraphQLNonNull(GraphQLString),
     },
   },
 });
