diff --git a/node_modules/@docusaurus/core/lib/client/serverEntry.js b/node_modules/@docusaurus/core/lib/client/serverEntry.js
index 5b766f8..b7f12c9 100644
--- a/node_modules/@docusaurus/core/lib/client/serverEntry.js
+++ b/node_modules/@docusaurus/core/lib/client/serverEntry.js
@@ -112,6 +112,6 @@ export default async function render(locals) {
     removeScriptTypeAttributes: true,
     removeStyleLinkTypeAttributes: true,
     useShortDoctype: true,
-    minifyJS: true,
+    minifyJS: false,
   });
 }
