diff --git a/node_modules/@graphql-mesh/fusion-runtime/dist/index.cjs b/node_modules/@graphql-mesh/fusion-runtime/dist/index.cjs
index 1699c3e..bd5efbf 100644
--- a/node_modules/@graphql-mesh/fusion-runtime/dist/index.cjs
+++ b/node_modules/@graphql-mesh/fusion-runtime/dist/index.cjs
@@ -14,7 +14,7 @@ var transportCommon = require('@graphql-mesh/transport-common');
 
 async function defaultTransportsGetter(kind) {
   try {
-    let transport = await import(`@graphql-mesh/transport-${kind}`);
+    let transport = require(`@graphql-mesh/transport-${kind}`);
     if (!transport) {
       throw new Error(
         `@graphql-mesh/transport-${kind} module does not export anything`
