diff --git a/node_modules/@graphql-mesh/fusion-runtime/dist/index.cjs b/node_modules/@graphql-mesh/fusion-runtime/dist/index.cjs
index 5831df0..a8fdb26 100644
--- a/node_modules/@graphql-mesh/fusion-runtime/dist/index.cjs
+++ b/node_modules/@graphql-mesh/fusion-runtime/dist/index.cjs
@@ -12,9 +12,9 @@ var stitchingDirectives = require('@graphql-tools/stitching-directives');
 var wrap = require('@graphql-tools/wrap');
 var transportCommon = require('@graphql-mesh/transport-common');
 
-async function defaultTransportsGetter(kind) {
+function defaultTransportsGetter(kind) {
   try {
-    let transport = await import(`@graphql-mesh/transport-${kind}`);
+    let transport = require(`@graphql-mesh/transport-${kind}`);
     if (!transport) {
       throw new Error(
         `@graphql-mesh/transport-${kind} module does not export anything`
