diff --git a/node_modules/@graphql-tools/stitching-directives/esm/stitchingDirectivesValidator.js b/node_modules/@graphql-tools/stitching-directives/esm/stitchingDirectivesValidator.js
index 5dea8fe..1eda1be 100644
--- a/node_modules/@graphql-tools/stitching-directives/esm/stitchingDirectivesValidator.js
+++ b/node_modules/@graphql-tools/stitching-directives/esm/stitchingDirectivesValidator.js
@@ -4,6 +4,7 @@ import { defaultStitchingDirectiveOptions } from './defaultStitchingDirectiveOpt
 import { parseMergeArgsExpr } from './parseMergeArgsExpr.js';
 const dottedNameRegEx = /^[_A-Za-z][_0-9A-Za-z]*(.[_A-Za-z][_0-9A-Za-z]*)*$/;
 export function stitchingDirectivesValidator(options = {}) {
+    return a => a;
     const { keyDirectiveName, computedDirectiveName, mergeDirectiveName, pathToDirectivesInExtensions, } = {
         ...defaultStitchingDirectiveOptions,
         ...options,
