diff --git a/node_modules/@theguild/prettier-config/index.js b/node_modules/@theguild/prettier-config/index.js
index d96bca4..9f95738 100644
--- a/node_modules/@theguild/prettier-config/index.js
+++ b/node_modules/@theguild/prettier-config/index.js
@@ -1,5 +1,5 @@
 // eslint-disable-next-line import/no-default-export
-export default {
+module.exports = {
   trailingComma: 'all', // default to `all` in v3
   printWidth: 100,
   singleQuote: true,
