diff --git a/node_modules/@theguild/components/dist/mermaid.mjs b/node_modules/@theguild/components/dist/mermaid.mjs
index 1cd6cb2..473e3b4 100644
--- a/node_modules/@theguild/components/dist/mermaid.mjs
+++ b/node_modules/@theguild/components/dist/mermaid.mjs
@@ -1,7 +1,7 @@
 // src/mermaid.tsx
 import { useEffect, useState } from "react";
 import mermaid from "mermaid";
-import { useTheme } from "nextra-theme-docs";
+import { useTheme } from 'next-themes';
 import { jsx } from "react/jsx-runtime";
 var id = 0;
 var Mermaid = ({ chart }) => {
