diff --git a/node_modules/bob-the-bundler/dist/commands/build.js b/node_modules/bob-the-bundler/dist/commands/build.js
index e1c8d4e..c8b4338 100644
--- a/node_modules/bob-the-bundler/dist/commands/build.js
+++ b/node_modules/bob-the-bundler/dist/commands/build.js
@@ -87,7 +87,7 @@ async function buildSingle({ distDir, distPath = '' }) {
     // generates
     const commonOutputOptions = {
         preferConst: true,
-        sourcemap: true,
+        sourcemap: false,
     };
     const generates = [
         {
@@ -131,6 +131,7 @@ async function build({ packagePath, cwd, pkg, fullName, config, reporter, distDi
     const inputOptions = {
         input: inputFile,
         plugins: [
+            require('@rollup/plugin-json')(),
             plugin_node_resolve_1.default(),
             auto_external_1.autoExternal({
                 packageJSONPath: packagePath,
@@ -148,7 +149,7 @@ async function build({ packagePath, cwd, pkg, fullName, config, reporter, distDi
     // generates
     const commonOutputOptions = {
         preferConst: true,
-        sourcemap: true,
+        sourcemap: false,
     };
     const generates = [
         {
@@ -178,6 +179,7 @@ async function build({ packagePath, cwd, pkg, fullName, config, reporter, distDi
             const inputOptions = {
                 input: binPath,
                 plugins: [
+                    require('@rollup/plugin-json')(),
                     plugin_node_resolve_1.default(),
                     auto_external_1.autoExternal({
                         packageJSONPath: packagePath,
@@ -191,7 +193,7 @@ async function build({ packagePath, cwd, pkg, fullName, config, reporter, distDi
                 preferConst: true,
                 sourcemap: options.sourcemap,
                 file: path_1.join(bobProjectDir, pkg.bin[alias].replace(`${exports.DIST_DIR}/`, "")),
-                format: "cjs",
+                format: pkg.bin[alias].endsWith('mjs') ? 'esm' : "cjs",
             });
         }));
     }
