diff --git a/node_modules/cosmiconfig/dist/loaders.js b/node_modules/cosmiconfig/dist/loaders.js
index 43126af..0995a74 100644
--- a/node_modules/cosmiconfig/dist/loaders.js
+++ b/node_modules/cosmiconfig/dist/loaders.js
@@ -1,29 +1,21 @@
-"use strict";
+'use strict';
 
-Object.defineProperty(exports, "__esModule", {
-  value: true
+Object.defineProperty(exports, '__esModule', {
+  value: true,
 });
 exports.loaders = void 0;
 
 /* eslint-disable @typescript-eslint/no-require-imports */
-let importFresh;
+const importFresh = require('import-fresh');
 
 const loadJs = function loadJs(filepath) {
-  if (importFresh === undefined) {
-    importFresh = require('import-fresh');
-  }
-
   const result = importFresh(filepath);
   return result;
 };
 
-let parseJson;
+const parseJson = require('parse-json');
 
 const loadJson = function loadJson(filepath, content) {
-  if (parseJson === undefined) {
-    parseJson = require('parse-json');
-  }
-
   try {
     const result = parseJson(content);
     return result;
@@ -33,16 +25,12 @@ const loadJson = function loadJson(filepath, content) {
   }
 };
 
-let yaml;
+const yaml = require('yaml');
 
 const loadYaml = function loadYaml(filepath, content) {
-  if (yaml === undefined) {
-    yaml = require('yaml');
-  }
-
   try {
     const result = yaml.parse(content, {
-      prettyErrors: true
+      prettyErrors: true,
     });
     return result;
   } catch (error) {
@@ -54,7 +42,7 @@ const loadYaml = function loadYaml(filepath, content) {
 const loaders = {
   loadJs,
   loadJson,
-  loadYaml
+  loadYaml,
 };
 exports.loaders = loaders;
 //# sourceMappingURL=loaders.js.map
