diff --git a/node_modules/disposablestack/AsyncDisposableStack/implementation.js b/node_modules/disposablestack/AsyncDisposableStack/implementation.js
index a4d1f6b..0e3a472 100644
--- a/node_modules/disposablestack/AsyncDisposableStack/implementation.js
+++ b/node_modules/disposablestack/AsyncDisposableStack/implementation.js
@@ -80,7 +80,7 @@ CreateMethodProperty(AsyncDisposableStack.prototype, 'disposeAsync', function di
 			DisposeResources(SLOT.get(asyncDisposableStack, '[[DisposeCapability]]'), NormalCompletion())
 		),
 		function (completion) {
-			return completion['?'](); // step 5
+			return completion?.['?'](); // step 5
 		}
 	);
 });
