diff --git a/node_modules/graphql-language-service-utils/dist/validateWithCustomRules.js b/node_modules/graphql-language-service-utils/dist/validateWithCustomRules.js
index e851b44..fa28b46 100644
--- a/node_modules/graphql-language-service-utils/dist/validateWithCustomRules.js
+++ b/node_modules/graphql-language-service-utils/dist/validateWithCustomRules.js
@@ -2,10 +2,9 @@
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.validateWithCustomRules = void 0;
 const graphql_1 = require("graphql");
-const ExecutableDefinitions_1 = require("graphql/validation/rules/ExecutableDefinitions");
 function validateWithCustomRules(schema, ast, customRules, isRelayCompatMode) {
     const rules = graphql_1.specifiedRules.filter(rule => {
-        if (rule === graphql_1.NoUnusedFragmentsRule || rule === ExecutableDefinitions_1.ExecutableDefinitions) {
+        if (rule === graphql_1.NoUnusedFragmentsRule || rule === graphql_1.ExecutableDefinitionsRule) {
             return false;
         }
         if (isRelayCompatMode && rule === graphql_1.KnownFragmentNamesRule) {
diff --git a/node_modules/graphql-language-service-utils/esm/validateWithCustomRules.js b/node_modules/graphql-language-service-utils/esm/validateWithCustomRules.js
index a8d6673..cd5120f 100644
--- a/node_modules/graphql-language-service-utils/esm/validateWithCustomRules.js
+++ b/node_modules/graphql-language-service-utils/esm/validateWithCustomRules.js
@@ -1,8 +1,7 @@
-import { Kind, specifiedRules, validate, NoUnusedFragmentsRule, KnownFragmentNamesRule, } from 'graphql';
-import { ExecutableDefinitions } from 'graphql/validation/rules/ExecutableDefinitions';
+import { Kind, specifiedRules, validate, NoUnusedFragmentsRule, KnownFragmentNamesRule, ExecutableDefinitionsRule } from 'graphql';
 export function validateWithCustomRules(schema, ast, customRules, isRelayCompatMode) {
     const rules = specifiedRules.filter(rule => {
-        if (rule === NoUnusedFragmentsRule || rule === ExecutableDefinitions) {
+        if (rule === NoUnusedFragmentsRule || rule === ExecutableDefinitionsRule) {
             return false;
         }
         if (isRelayCompatMode && rule === KnownFragmentNamesRule) {
