diff --git a/node_modules/guild-docs/esm/mermaid.js b/node_modules/guild-docs/esm/mermaid.js
index 0c0a96a..f24e85b 100644
--- a/node_modules/guild-docs/esm/mermaid.js
+++ b/node_modules/guild-docs/esm/mermaid.js
@@ -1,6 +1,10 @@
 import React, { useEffect, useState } from 'react';
 import mermaid from 'mermaid';
-import { useTheme } from '@theguild/components';
+// Got error while running `next build`
+// > Build error occurred
+// Error [ERR_MODULE_NOT_FOUND]: Cannot find package '/Users/dimitri/Desktop/GUILD/graphql-mesh/node_modules/@theguild/components/'
+// imported from /Users/dimitri/Desktop/GUILD/graphql-mesh/node_modules/guild-docs/esm/mermaid.js
+import { useTheme } from 'next-themes';
 /**
  * Assign a unique ID to each mermaid svg as per requirements of `mermaid.render`.
  */
