diff --git a/node_modules/uWebSockets.js/uws.js b/node_modules/uWebSockets.js/uws.js
index e6a5770..038a6a2 100644
--- a/node_modules/uWebSockets.js/uws.js
+++ b/node_modules/uWebSockets.js/uws.js
@@ -15,10 +15,15 @@
  * limitations under the License.
  */

+const isDebug = ['1', 'y', 'yes', 't', 'true'].includes(String(process.env.DEBUG));
+
 module.exports = (() => {
 	try {
 		return require('./uws_' + process.platform + '_' + process.arch + '_' + process.versions.modules + '.node');
 	} catch (e) {
+    if (isDebug) {
+      console.error(e);
+    }
 		throw new Error('This version of uWS.js supports only Node.js versions 18, 20, 21 and 22 on (glibc) Linux, macOS and Windows, on Tier 1 platforms (https://github.com/nodejs/node/blob/master/BUILDING.md#platform-list).\n\n' + e.toString());
 	}
 })();

